developers/weaviate/concepts/data.md

```diff
@@ -207,12 +206,46 @@ Sharding has several benefits:
 - Fast, efficient querying
 - Easy and robust setup and clean up
 
-Starting in `v1.20`, shards are more lightweight. You can easily have 50,000, or more, active shards per node. This means that you can support 1M concurrently active tenants with just 20 or so nodes.
-
-Starting in `v1.20.1`, you can specify tenants as active (`HOT`) or inactive (`COLD`). For more details on managing tenants, see [Multi-tenancy operations](../manage-data/multi-tenancy.md).
+Tenant shards are more lightweight. You can easily have 50,000, or more, active shards per node. This means that you can support 1M concurrently active tenants with just 20 or so nodes.
 
 Multi-tenancy is especially useful when you want to store data for multiple customers, or when you want to store data for multiple projects.
 
+### Tenant status
+
+:::info Multi-tenancy availability
+- Tenant activity status setting added in `v1.21`
+- `OFFLOADED` status added in `v1.26`
+:::
+
+Tenants can be `ACTIVE`, `INACTIVE`, `OFFLOADED`, `OFFLOADING`, or `ONLOADING`.
+
+- `ACTIVE` tenants are loaded and available for read and write operations.
+- In all other states, the tenant is not available for read or write access. Access attempts return an error message.
+    - `INACTIVE` tenants are stored on local disk storage for quick activation.
+    - `OFFLOADED` tenants are stored on cloud storage. This status is useful for long-term storage for tenants that are not frequently accessed.
+    - `OFFLOADING` tenants are being moved to cloud storage. This is a transient status, and therefore not user-specifiable.
+    - `ONLOADING` tenants are being loaded from cloud storage. This is a transient status, and therefore not user-specifiable. An `ONLOADING` tenant may be being warmed to a `ACTIVE` status or a `INACTIVE` status.
+
+For more details on managing tenants, see [Multi-tenancy operations](../manage-data/multi-tenancy.md).
+
+| Status | Available | Description | User-specifiable |
+| :-- | :-- | :-- | :-- |
+| `ACTIVE` | Yes | Loaded and available for read/write operations. | Yes |
+| `INACTIVE` | No | On local disk storage, no read / write access. Access attempts return an error message. | Yes |
+| `OFFLOADED` | No | On cloud storage, no read / write access. Access attempts return an error message. | Yes |
+| `OFFLOADING` | No | Being moved to cloud storage, no read / write access. Access attempts return an error message. | No |
+| `ONLOADING` | No | Being loaded from cloud storage, no read / write access. Access attempts return an error message. | No |
+
+:::info Tenant status renamed in `v1.26`
+In `v1.26`, the `HOT` status was renamed to `ACTIVE` and the `COLD` status was renamed to `INACTIVE`.
+:::
+
+#### Frozen (offloaded) tenants
```

Comment:
**suggestion:** shall we call it `Offloaded (frozen) tenants` to not confuse the users maybe we avoid the old renaming ? 

i know that the naming just got changed

Reply:
Oh yeah good idea.